site/en/blog/timer-throttling/index.md

```diff
@@ -0,0 +1,238 @@
+behavior, but timers are often used where a different API would be more
+efficient, and more reliable.
+
+Ok, that was pretty jargon heavy, and a bit ambiguous. Let's dig in…
+
+## Terminology
+
+### Hidden pages
+
+Generally, 'hidden' means a different tab is active, or the window has been
+minimized, but browsers may consider a page hidden whenever its content is
+totally not-visible. Some browsers go further than others here, but you can
+always use the [page visibility
+API](https://developer.mozilla.org/en-US/docs/Web/API/Page_Visibility_API) to
+track when the browser thinks visibility has changed.
+
+### JavaScript timers
+
+By "JavaScript timers" I mean `setTimeout` and `setInterval`, which allow you to
+schedule a callback sometime in the future. Timers are useful, and they aren't
+going away, but sometimes they're used to poll state when an event would be more
+efficient, and more accurate.
+
+### Chained timers
+
+If you call `setTimeout` in the same task as a `setTimeout` callback, the second
+invocation is 'chained'. With `setInterval`, each iteration is part of the
+chain. This might be easier to understand with code:
+
+```js
+let chainCount = 0;
+
+setInterval(() => {
+  chainCount++;
+  console.log(`This is number ${chainCount} in the chain`);
+}, 500);
+```
+
+And:
+
+```js
+let chainCount = 0;
+
+function setTimeoutChain() {
+  setTimeout(() => {
+    chainCount++;
+    console.log(`This is number ${chainCount} in the chain`);
+    setTimeoutChain();
+  }, 500);
+}
+```
+
+Ok, I think that's all the terms you need to know.
+
+## How the throttling works
+
+The throttling happens in stages:
+
+### Minimal throttling
+
+This happens to timers that are scheduled when _any_ of the following is true:
+
+- The page is visible.
+- The page has made noises in the past 30 seconds. This can be from any of the
+  sound-making APIs, but a silent audio track doesn't count.
+
+The timer isn't throttled, unless the requested timeout is less than 4ms, and
+the chain count is 5 or greater, in which case the timeout is set to 4ms. This
+isn't new, browsers have done this for many years.
+
+### Throttling
+
+This happens to timers that are scheduled when 'Minimal throttling' doesn't
+apply, and _any_ of the following is true:
+
+- The chain count is less than 5.
+- The page has been hidden for less than 5 minutes.
+- WebRTC is in use. Specifically, there's an `RTCPeerConnection` with an 'open'
+  `RTCDataChannel` or a 'live' `MediaStreamTrack`.
+
+The browser will check timers in this group once per **second**. Because they're
+only checked once per second, timers with a similar timeout will batch together,
+consolidating the time the tab needs to run code. This isn't new either,
+browsers have been doing this to some extent for years.
+
+### Intensive throttling
+
+Ok, this is the new bit in Chrome 88. This happens to timers that are scheduled
+when neither 'Minimal throttling' or 'Throttling' apply, meaning _all_ of the
+following is true:
```

Comment:
```suggestion
OK, here's the new bit in Chrome 88. Intensive throttling happens to timers that are
scheduled when none of the **minimal throttling** or **throttling** conditions
apply, and *all* of the following conditions are true:
```
Just a slight rephrasing to attempt to increase clarity

Reply:
I'll add this but with italic rather than bold